src/index.html

```diff
@@ -7,12 +7,67 @@
       content="width=device-width, initial-scale=1.0"
     />
     <title>Product cards</title>
+
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet"
+    />
+
     <link
       rel="stylesheet"
       href="./styles/index.scss"
     />
   </head>
   <body>
-    <h1>Product cards</h1>
+    <div
+      class="card"
+      data-qa="card"
+    >
+      <img
+        class="card__image"
+        src="images/imac.jpeg"
+        alt="iMac"
```

Comment:
That's already better, but try to give a more extensive alt. It can be a whole name or a product description.
This is done for many things:
1. When the image does not load, so that the user understands what kind of image it is.
2. For voice assistants, so that people who are blind can hear what the picture is.
3. And for SEO optimization.